pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -318,6 +318,8 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 	found := true
 	writer := newLogWriter(stdout, stderr, opts)
 	msg := &logMessage{}
+	baseName := filepath.Base(path)
+	dir := filepath.Dir(path)
```

Comment:
nit: 
```suggestion
	baseName, dir := filepath.Split(path)
```

Reply:
I don't think we usually edit cherry-pick PRs.

Reply:
/lgtm
then